### ISSUE TITLE
Max-width should be applied to paragraphs

### ISSUE THREAD

**Issue by jrbarnes9** [ONSdigital/design-system#2443]
`patternlib.scss` includes styles to add `max-width` to paragraphs, lists and headings within the page content, to meet [WCAG guidelines for readability ](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-visual-presentation.html#:~:text=Lines%20should%20not%20exceed%2080,writing%20system%20for%20the%20text.)

This should be applied to the core design system, either as an option for services, or as a default to ensure the width of a line of content can never exceed 80 characters.

**Comment by yatesn** [comment]
Scope may have changed due to service manual work - to be reviewed

**Comment by yatesn** [comment]
Move to service-manual repo
